src/components/WindowTopBar.js

```diff
@@ -28,32 +27,32 @@ export class WindowTopBar extends Component {
       maximizeWindow, maximized, minimizeWindow, focused,
     } = this.props;
     return (
-      <AppBar position="relative">
-        <Toolbar disableGutters className={classNames(classes.windowTopBarStyle, focused ? classes.focused : null, ns('window-top-bar'))} variant="dense">
```

Comment:
I'm not sure I understand why this PR removes the mirador-namespaces CSS classes; I was under the impression that those classes was our preference to support some downstream theming use cases?

Reply:
Namespaced css classes are bloat.  Prefer providing class references as theme variables instead.  Also. Integration tests need not depend on class selectors with jss.